File: MomoQR/app/src/main/java/github/daisukiKaffuChino/MomoQR/logic/bean/FavBean.java
package github.daisukiKaffuChino.MomoQR.logic.bean;

import androidx.lifecycle.LiveData;

public class FavBean extends LiveData<FavBean> {
    private String content;
    private String id;
    private String title;
    private String img;
    private long time;
    private boolean isImportant;

    public String getId() {
        return this.id;
    }

    public void setId(String str) {
        this.id = str;
        postValue(this);
    }

    public String getContent() {
        return this.content;
    }

    public void setContent(String str) {
        this.content = str;
        postValue(this);
    }

    public String getImg() {
        return this.img;
    }

    public void setImg(String str) {
        this.img = str;
        postValue(this);
    }

    public long getTime() {
        return this.time;
    }

    public void setTime(long i) {
        this.time = i;
        postValue(this);
    }

    public String getTitle() {
        return this.title;
    }

    public void setTitle(String str) {
        this.title = str;
        postValue(this);
    }

    public boolean getIsImportant() {
        return this.isImportant;
    }

    public void setIsImportant(boolean isImportant) {
        this.isImportant = isImportant;
        postValue(this);
    }
}

File: app/src/main/java/github/daisukiKaffuChino/MomoQR/logic/bean/FavBean.kt
package github.daisukiKaffuChino.MomoQR.logic.bean

import androidx.lifecycle.LiveData

class FavBean : LiveData<FavBean?>() {

    var content: String? = null
        set(str) {
            field = str
            postValue(this)
        }
    var id: String? = null
        set(str) {
            field = str
            postValue(this)
        }
    var title: String? = null
        set(str) {
            field = str
            postValue(this)
        }
    var img: String? = null
        set(str) {
            field = str
            postValue(this)
        }
    var time: Long = 0
        set(i) {
            field = i
            postValue(this)
        }
    var star = false
        set(star) {
           field = star
            postValue(this)
        }
}
